docs/stencil-docs/installing-stencil-cli/installing-stencil.md

```diff
@@ -48,29 +48,38 @@ nvm use 12
 # Install Stencil CLI
 npm install -g @bigcommerce/stencil-cli
 ```
+
+### Installing on Mac with M1 chip
+To install Stencil CLI and its dependencies on Mac with a M1 chip, open a Rosetta terminal and run the following commands.
+
 <div class="HubBlock--callout">
 <div class="CalloutBlock--info">
 <div class="HubBlock-content">
 
 <!-- theme: info -->
 
-> Use these instructions for Mac's with a M1 chip.
+> These instructions have been tested on a **MacBook Air** with a M1 chip.
 
 </div>
 </div>
 </div>
 
-```shell
 
-# Install Node Version Manager (nvm)
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
+Open terminal using Rosetta.

```

Comment:
Should these be numbered lists since they are sequential?

Reply:
Bumping this. Should the line above also be numbered?